ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -59,4 +60,54 @@ public struct ShadowInfo: Equatable {
 
     /// The vertical offset of the shadow for shadow 2.
     public let yTwo: CGFloat
+
+    /// The number that the figma blur needs to be divided by to properly display shadows
+    private let shadowBlurAdjustment: CGFloat = 0.5
+}
+
+public class ShadowUtil {
+
+    public static func applyShadow(_ shadowInfo: ShadowInfo, for view: UIView) {
```

Comment:
Question: could this just be an extension on `ShadowInfo` to eliminate the need for a new class? So instead of calling 

```swift
ShadowUtil.applyShadow(shadowInfo, for: view)
```

you could just call

```swift
shadowInfo.apply(to: view)
```